### PR TITLE
MacOS builds added into Github actions

### PR DESCRIPTION
**Pull request by Ecostack**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -63,3 +63,55 @@ jobs:
...
+      - name: Upload binaries
+        uses: svenstaro/upload-release-action@v2
+        with:
+          repo_token: ${{ secrets.GITHUB_TOKEN }}
+          file: tiny-world.tar.gz
+          asset_name: tiny-world-${{ env.RELEASE_VERSION }}-macos-arm64.tar.gz
+          tag: ${{ github.ref }}
+
+  build-macos-amd64:
+    name: MacOS adm64 build
```
Comment (mlange-42) [inline .github/workflows/release.yml:94]:
```suggestion
    name: MacOS amd64 build
```

**Comment by mlange-42** [comment]
@Ecostack Many thanks!

Do you have the opportunity to test the MacOS build artifacts locally? See [this job](https://github.com/mlange-42/tiny-world/actions/runs/8234802787).

**Comment by Ecostack** [comment]
Let me check this today

**Comment by Ecostack** [comment]
Where do you see `GNUSparseFile.0`?
If I download the tar.gz, this is what I get upon extraction:

Screenshot 2024-03-12 at 10 28 41

**Comment by Ecostack** [comment]
I will look into creating a proper .app file instead of just a CLI. Though, I cannot find the GNUSparseFile

**Pull request merged**